make/source.mk

```diff
@@ -333,6 +333,11 @@ endif #!F1
 # check if target.mk supplied
 SRC := $(STARTUP_SRC) $(MCU_COMMON_SRC) $(TARGET_SRC) $(VARIANT_SRC)
 
+# Files that should not be optimized, useful for debugging IMPRECISE cpu faults.
+# Specify FULL PATH, e.g. "./lib/main/STM32F7/Drivers/STM32F7xx_HAL_Driver/Src/stm32f7xx_ll_sdmmc.c"
+NOT_OPTIMISED_SRC := $(NOT_OPTIMISED_SRC) \
+#    ./lib/main/STM32F7/Drivers/STM32F7xx_HAL_Driver/Src/stm32f7xx_hal_sd.c \
```

Comment:
This should be removed. The rest looks fine. Also, n.b. this also works when used on the command line, e.g.

```
make MATEKF722SE NOT_OPTIMISED_SRC=./src/main/drivers/usb_io.c
```

Reply:
done